src/App.tsx

```diff
@@ -1,156 +1,75 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Consider removing the 'eslint-disable jsx-a11y/control-has-associated-label' comment since it's not needed anymore.